samples/core/enumopencl/main.cpp

```diff
@@ -67,6 +67,9 @@ static cl_int PrintDeviceInfoSummary(const std::vector<cl::Device> devices)
 
 int main(int argc, char** argv)
 {
+    (void)argc;
+    (void**)argv;
```

Comment:
```suggestion
    (void)argv;
```

While your suggested change is not wrong just alway cast to `(void)` is the idiomatic way to make this as unused.